templates/email_verification.html

```diff
@@ -0,0 +1,61 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Email verification</title>
+    <style>
+        .container {
+            padding: 20px;
+        }
+
+        .divider {
+            width: 58%;
+            border-top: 1px solid #ccc;
+        }
+        .button {
+          padding: 10px 20px;
+          font-weight: bold;
+          background-color: #007bff;
+          color: #fff;
+          text-decoration: none;
+            border-radius: 5px;
+        }
+        .iconify {
+            width: 7rem;
+            height: 7rem;
+            color: rgba(0, 123, 255, 0.92);
+        }
+        .accept-top{
+            padding-bottom: 2rem;
+        }
+        .accept-bottom{
+            padding-top: 2rem;
+            padding-bottom: 3rem;
+        }
+        p{
+            font-family: sans-serif;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <div>
+            <span class="iconify" data-icon="line-md:email"></span>
+        </div>
+    <div class="divider"></div>
+    <h1>Hello {{ email}}!</h1>
+
+    <div>
+        <p>Please verify your email address by clicking the button below</p>
+        <a href="{{ verification_url }}" class="button">Verify Email Address</a>
```

Comment:
The `verification_url` should also be provided as text as well. Maybe another line underneath so users can copy and paste